system/lib/libcxx/include/__config

```diff
@@ -1160,10 +1172,6 @@ _LIBCPP_FUNC_VIS extern "C" void __sanitizer_annotate_contiguous_container(
        _LIBCPP_HAS_NO_THREADS is defined.
 #endif
 
-#if defined(__STDCPP_THREADS__) && defined(_LIBCPP_HAS_NO_THREADS)
-#error _LIBCPP_HAS_NO_THREADS cannot be set when __STDCPP_THREADS__ is set.
-#endif
```

Comment:
why remove this? if it's a local change, perhaps `#if 0` with a comment?

Reply:
Sorry I forgot to mention, this was removed upstream already in https://reviews.llvm.org/D92349